week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -23,8 +23,8 @@
 function checkCodeIsThere(stringText) {
   let magicWord = "code";
   //edit code below
-  if (stringText) {
-    return stringText;
+  if (stringText.includes("code")) {
+    return stringText.indexOf("code");
   } else {
```

Comment:
In this function you can use the `magicWord` variable. Remember to keep your code DRY!